pkg/metadata/host/host_test.go

```diff
@@ -18,6 +20,8 @@ import (
 
 func TestGetPayload(t *testing.T) {
 	p := GetPayload("myhostname")
+	pj, _ := json.Marshal(p)
+	fmt.Println(string(pj))
```

Comment:
Is this just testing that the payload is unmarshalled correctly? Capturing the error and using `assert.NoError()` instead of letting it panic would give better output I think.

Reply:
oups good catch, this was for debugging purposes I'll remove it 🙈